cc/tflite_micro/tflite_micro.cc

```diff
@@ -22,20 +22,24 @@
 #include "tensorflow/lite/micro/micro_interpreter.h"
 #include "tensorflow/lite/schema/schema_generated.h"
 
-TfLiteTensor* input = nullptr;
-TfLiteTensor* output = nullptr;
+// Max number of input/output tensors
+#define MAX_TENSORS 100
```

Comment:
Is having multiple input tensors specific to this demo?
Or it's a general use-case that we need to support for other models?

Reply:
I think it's generic and we should support as many models as possible in a generic way as we can.